modules/rostests/kmtests/ntos_mm/MmSection.c

```diff
@@ -224,45 +269,64 @@ TestCreateSection(
         ok_eq_longlong(MaximumSize.QuadPart, 1LL);
         ok(SectionObject != KmtInvalidPointer, "Section object pointer untouched\n");
         ok(SectionObject != NULL, "Section object pointer NULL\n");
+        if (GetCurrentNTVersion() >= _WIN32_WINNT_WIN8)
+            PointerCount2 -= 2;
         CheckObject(FileHandle2, PointerCount2, 1L);
-        CheckSection(SectionObject, SEC_IMAGE);
+        if (GetCurrentNTVersion() < _WIN32_WINNT_WIN8)
+            CheckSection(SectionObject, SEC_IMAGE);
         TestMapView(SectionObject, FALSE, TRUE);
 
         if (SectionObject && SectionObject != KmtInvalidPointer)
             ObDereferenceObject(SectionObject);
 
+        if (GetCurrentNTVersion() >= _WIN32_WINNT_WIN8)
+            PointerCount2--;
         CheckObject(FileHandle2, PointerCount2, 1L);
         SectionObject = KmtInvalidPointer;
         MaximumSize.QuadPart = 1;
         KmtStartSeh()
             Status = MmCreateSection(&SectionObject, 0, NULL, &MaximumSize, PAGE_READONLY, SEC_IMAGE, NULL, FileObject2);
         KmtEndSeh(STATUS_SUCCESS);
-        ok_eq_hex(Status, STATUS_SUCCESS);
+        if (GetCurrentNTVersion() < _WIN32_WINNT_WIN8)
+            ok_eq_hex(Status, STATUS_SUCCESS);
+        else
+            ok_eq_hex(Status, STATUS_INVALID_PARAMETER_6);
         ok_eq_longlong(MaximumSize.QuadPart, 1LL);
-        ok(SectionObject != KmtInvalidPointer, "Section object pointer untouched\n");
+        if (GetCurrentNTVersion() < _WIN32_WINNT_WIN8)
+            ok(SectionObject != KmtInvalidPointer, "Section object pointer untouched\n");
         ok(SectionObject != NULL, "Section object pointer NULL\n");
-        //++PointerCount2;
+        if (GetCurrentNTVersion() == _WIN32_WINNT_WS03)
+            ++PointerCount2;
+        else if (GetCurrentNTVersion() >= _WIN32_WINNT_WIN8)
+            PointerCount2--;
```

Comment:
Nit:
```suggestion
            --PointerCount2;
```
or, `PointerCount2++;` for the other one :joy: